app/managers.py

```diff
@@ -10,10 +10,10 @@ def __init__(self) -> None:
 
     def _create_table(self) -> None:
         self._connection.execute(
-            f"CREATE TABLE IF NOT EXISTS {self.table_name} ("
-            "id INTEGER PRIMARY KEY AUTOINCREMENT, "
-            "first_name TEXT NOT NULL, "
-            "last_name TEXT NOT NULL"
+            f"""CREATE TABLE IF NOT EXISTS {self.table_name} ("""
+            """id INTEGER PRIMARY KEY AUTOINCREMENT, """
+            """first_name TEXT NOT NULL, """
+            """last_name TEXT NOT NULL"""

```

Comment:
Olia meant that instead of adding "" in each line, you can use triple quotes at the beginning and end of the sql query like
`"""
SELECT * FROM users

where id = 4
"""`
It's like a multi-line comment